lib/apiForSheets.js

```diff
@@ -95,6 +95,10 @@ module.exports.generateOutput = async function (rows, prNumber, prDetails) {
       'https://github.com/oppia/oppia/wiki/Contributing-code-to-Oppia#' +
```

Comment:
The tests need to be updated following this change.

Reply:
working on tests

Reply:
tests updated.